pkg/controller/controller_binding.go

```diff
@@ -712,7 +712,7 @@ func (c *controller) injectServiceInstanceCredential(binding *v1alpha1.ServiceIn
 		err = nil
 
 		// Create new secret
-		secret := &v1.Secret{
+		secret := &apiv1.Secret{
```

Comment:
interesting that api and v1 are combined.

Reply:
@MHBauer I will rename all occurences of this package to the alias `corev1` while doing rebase onto 1.8 to be clear.